polkadot/runtime/parachains/src/paras_inherent/tests.rs

```diff
@@ -145,6 +183,122 @@ mod enter {
 				Pallet::<Test>::on_chain_votes().unwrap().session,
 				2
 			);
+
+			assert_eq!(
+				inclusion::PendingAvailability::<Test>::get(ParaId::from(0))
+					.unwrap()
+					.into_iter()
+					.map(|c| c.core_occupied())
+					.collect::<Vec<_>>(),
+				vec![CoreIndex(0)]
+			);
+			assert_eq!(
+				inclusion::PendingAvailability::<Test>::get(ParaId::from(1))
+					.unwrap()
+					.into_iter()
+					.map(|c| c.core_occupied())
+					.collect::<Vec<_>>(),
+				vec![CoreIndex(1)]
+			);
+		});
+	}
+
+	#[test]
+	// ParaId 1 has one pending candidate on core 0.
+	// ParaId 2 has one pending candidate on core 1.
+	// ParaId 3 has three pending candidates on cores 2, 3 and 4.
+	// All of them are being made available in this block. Propose 5 more candidates (one for each
+	// core) and check that they're successfully backed and the old ones enacted.
+	fn include_backed_candidates_elastic_scaling() {
+		let config = default_config();
+		assert!(config.configuration.config.scheduler_params.lookahead > 0);
+
+		new_test_ext(config).execute_with(|| {
+			// Set the elastic scaling MVP feature.
+			<configuration::Pallet<Test>>::set_node_feature(
```

Comment:
Isn't this set by default already?

Reply:
not in unit tests. it's being set here: https://github.com/paritytech/polkadot-sdk/blob/5fd72a1f5e7f4db02b4962fb146e4b8d154a485c/polkadot/node/service/src/chain_spec.rs#L161

Reply:
Maybe it should.

Reply:
If we add this, I'd have to fix a bunch of tests which make the assumption that it's not enabled by default. I wouldn't do that now